src/containers/inputContainer.js

```diff
@@ -74,7 +74,7 @@ function inputContainer(Input) {
           labelKey,
           onRemove,
           renderToken,
-          role: '',
+          role: 'combobox',
```

Comment:
As discussed on our other thread, the `combobox` role is not appropriate for the multi-select version. See code comment above re: https://www.w3.org/TR/wai-aria-practices-1.1/#combobox